.github/workflows/build_branch34.yml

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "Build (branch-3.4, Scala 2.13, Hadoop 3, JDK 8)"
+
+on:
+  schedule:
+    - cron: '0 9 * * *'
+
+jobs:
+  run-build:
+    permissions:
+      packages: write
+    name: Run
+    uses: ./.github/workflows/build_and_test.yml
+    if: github.repository == 'apache/spark'
+    with:
+      java: 8
+      branch: branch-3.4
+      hadoop: hadoop3
+      envs: >-
+        {
+          "SCALA_PROFILE": "scala2.13"
```

Comment:
Hm, wonder if we should use 2.13 though. Maybe let's use all the default ones for now?

Reply:
I guess this is the main purpose because `branch-3.4` is already tested at every commit with the default ones?

Reply:
Yes, this is a clone of [build_branch33.yml](https://github.com/apache/spark/blob/master/.github/workflows/build_branch33.yml), and build_branch33.yml also test Scala 2.13, do we need to change this to 2.12 now?

Reply:
Oh yeah.

Reply:
Sorry, I didn't get your opinion from `Oh yeah.` ...